.github/workflows/ci.yml

```diff
@@ -215,6 +214,22 @@ jobs:
           # was created in this workflow.
           tls_ca_cert: ${{ env.TLS_CA_CERT }}
 
+      - name: Cleanup out dir
+        run: sudo rm -rf out_repo
```

Comment:
The previous action run outputs a dir `out_repo`. It comes from the docker context, causing permission issues with the next test. We can remove it here.